.travis.yml

```diff
@@ -0,0 +1,34 @@
+sudo: required
+
+language: generic
+
+env:
+  - DOCKER_COMPOSE_VERSION=1.14.0
+
+install:
+  # Installing docker
+  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+  - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+  - sudo apt-get update
+  - sudo apt-get -y install docker-ce
+
+  # Installing docker-compose
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L "https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-$(uname -s)-$(uname -m)" > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+
+before_script:
+  # Check docker & docker-compose versions
+  - docker --version
+  - docker-compose --version
+
+script:
+  - docker-compose build
+  - docker-compose up -d
+
+  # Verifications
+  - sleep 4m
+  - docker network ls
```

Comment:
What's the point of the `docker network ls` command here?

Reply:
Just to see some extra information after starting the containers.

Actually 4 containers are shown.

Reply:
Didn't you mean `docker-compose ps` if you want to see the containers running?

Reply:
``docker-compose  ps`` can be also added to show more informations of running containers.
``docker network`` allows to show other kind of information

Reply:
It shows
```$ docker network ls
NETWORK ID          NAME                DRIVER              SCOPE
d7d3e6672af7        bridge              bridge              local
fa39c8877bbd        dockerelk_elk       bridge              local
3fd9b4092147        host                host                local
0617e0553039        none                null                local
```
Is that really helpful?

Reply:
I don't think so. No point to show running container either, the healthcheck is the only interesting thing here.

Reply:
I would get the point of showing containers in some situations. Occasionally one of them terminates during startup, because some configuration is deprecated or multiple other reasons. Displaying the state of the stack would at least explain why the healthcheck fails, and help us pinpoint the failing service. What do you think @deviantony ?

Reply:
What about integrating `docker-compose logs` instead?

Reply:
You're right, makes more sense.

Reply:
I think that `docker-compose logs` would be enough to diagnose in case of issue.

Reply:
Ok, if I undertand, I'll replace ``docker network`` by ``docker-compose logs``